internal/pkg/mining/worker.go

```diff
@@ -208,32 +204,31 @@ func (w *DefaultWorker) Mine(ctx context.Context, base block.TipSet, nullBlkCoun
 		outCh <- Output{Err: err}
 		return
 	}
-	// lookback consensus.ElectionLookback for the election ticket
-	baseHeight, err := base.Height()
+	baseEpoch, err := base.Height()
 	if err != nil {
 		log.Warnf("Worker.Mine couldn't read base height %s", err)
 		outCh <- Output{Err: err}
 		return
 	}
-	ancestors, err := w.getAncestors(ctx, base, baseHeight+(abi.ChainEpoch(nullBlkCount+1)))
-	if err != nil {
-		log.Warnf("Worker.Mine couldn't get ancestorst %s", err)
-		outCh <- Output{Err: err}
-		return
-	}
-	electionTicket, err := sampling.SampleNthTicket(int(miner.ElectionLookback-1), ancestors)
```

Comment:
To make this PR right I believe you should sample `electionRandomness` (no longer a ticket) and then pass that election randomness into the existing postProofGeneration code (modulo changing ticket type to randomness type).

There is another step maybe done by the sectorbuilder or maybe we have to do it that transforms the postRandomness VRF pi into a post challenge seed for use in the GenerateCandidates function.  We'll need to consult the spec but that transformation is at least another hash to get vrf digest and maybe it also mixes in epoch / other entropy.

cc @sternhenri to verify that the order of operations to get epost candidates is: 1) draw lookback randomness 2) create post randomness vrf Pi off of that randomness 3) draw randomness (vrf digest / maybe mising) from post randomness for generatecandidates.